job/server/src/main/java/alluxio/job/plan/load/LoadDefinition.java

```diff
@@ -54,6 +55,8 @@
   private static final Logger LOG = LoggerFactory.getLogger(LoadDefinition.class);
   private static final int MAX_BUFFER_SIZE = 500 * Constants.MB;
 
```

Comment:
nit: I would remove the empty line 57. We try to keep the src code concise and compact